### PR TITLE
Update troubleshooting-mobile.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added troubleshooting to this FAQ instead of to a separate page.

### CODE REVIEW

**Review by wiersgallak — COMMENTED** [review]
We removed server logs from Mattermost Cloud.  I think this section needs to be removed. @adamjclarkson Can you confirm we removed the logs page?

**Review by adamjclarkson — APPROVED** [review]

**Comment by justinegeffen** [comment]
@adamjclarkson, before I merge this can I confirm that the server content should/shouldn't be in the doc?

**Comment by adamjclarkson** [comment]
> @adamjclarkson, before I merge this can I confirm that the server content should/shouldn't be in the doc?

Confirmed, there are no server logs in the console, so server content should not be in the doc

File: source/cloud/cloud-mobile/troubleshooting-mobile.rst [hunk 1]
```diff
@@ -10,19 +10,33 @@ Confirm that your server URL has no typos and that it includes ``https://``.
...
+If you are seeing this message all the time, and your internet connection seems fine, ask your server administrator if the server uses NGINX or another webserver as a reverse proxy. If so, they should check that it is configured correctly for `supporting the websocket connection for APIv4 endpoints <https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server>`__.
 
 I’m not receiving push notifications on my device
 -------------------------------------------------
 
-Please see our documentation on `troubleshooting push notifications <https://docs.mattermost.com/cloud/cloud-mobile/troubleshooting-mobile.html>`_.
+If you did not receive a push notification when testing push notifications, use the following procedure to troubleshoot:
```
Comment (justinegeffen) [inline source/cloud/cloud-mobile/troubleshooting-mobile.rst:25]: @adamjclarkson, should this entire section be removed?

Comment (justinegeffen) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst:25]: @adamjclarkson, following up on this to confirm.

Comment (wiersgallak) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst:25]: @justinegeffen This does need to be removed.

Comment (justinegeffen) [reply source/cloud/cloud-mobile/troubleshooting-mobile.rst]: I've removed it.

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**